patches/master_patch.patch

```diff
@@ -1548,17 +1527,7 @@ index 85ece52a001652a3414a47189b2711f215a66290..960abaeab72bbf45b8b460fbbaf1cdb4
  // These are not documented, so use only after checking -respondsToSelector:.
  @interface NSApplication (UndocumentedSpeechMethods)
  - (void)speakString:(NSString*)string;
-@@ -455,7 +461,9 @@ float FlipYFromRectToScreen(float y, float rect_height) {
-   background_layer_.reset([[CALayer alloc] init]);
-   // Set the default color to be white. This is the wrong thing to do, but many
-   // UI components expect this view to be opaque.
-+#ifdef MUON_CHROMIUM_BUILD
-   [background_layer_ setBackgroundColor:CGColorGetConstantColor(kCGColorWhite)];
-+#endif
-   [cocoa_view_ setLayer:background_layer_];
-   [cocoa_view_ setWantsLayer:YES];
- 
-@@ -1856,7 +1864,14 @@ - (BOOL)acceptsMouseEventsWhenInactive {
+@@ -1863,7 +1869,14 @@ void RenderWidgetHostViewMac::OnDisplayMetricsChanged(
```

Comment:
Setting the background color to white was removed here: https://chromium.googlesource.com/chromium/src/+/e1979f60524da9dab9d8cc19a23910f8d5dcf180